src/prefect_server/graphql/flow_groups.py

```diff
@@ -26,6 +26,16 @@ async def resolve_set_flow_group_labels(
     return {"success": result}
 
 
+@mutation.field("set_flow_group_run_config")
+async def resolve_set_flow_group_run_config(
+    obj: Any, info: GraphQLResolveInfo, input: dict
+) -> dict:
+    result = await api.flow_groups.set_flow_group_run_config(
+        flow_group_id=input["flow_group_id"], run_config=input["run_config"]
```

Comment:
if `run_config` is intended to be optional, we'll need to account for that here
```suggestion
        flow_group_id=input["flow_group_id"], run_config=input.get("run_config")
```